android/.gitignore

```diff
@@ -41,6 +41,9 @@ KMEA/**/assets/resources
 # keymanweb OSK font
 KMEA/**/assets/keymanweb-osk.*
 
+# KMAPRo generated offline help
+KMAPro/**/info
```

Comment:
Is this just a single folder or two? If so, I think I'd prefer to have the full path for each one. Probably ditto on the other entries!

Reply:
It's a lot longer than two folders
`KMAPro\kMAPro\src\main\assets\info`

Maybe a middle ground of `KMAPro/**/assets/info`?